generate_fake.php

```diff
@@ -8,16 +8,15 @@
 
 require_once __DIR__.'/vendor/autoload.php';
 
-if ($argc < 3) {
-    echo 'You have to pass domain and step arguments.'.PHP_EOL;
+if ($argc !== 2) {
+    echo 'You have to pass too many arguments.'.PHP_EOL;
```

Comment:
Yeah, you're right .  This syntax is not readable so I use ```>``` to make syntax easy to understand.